analyticscloud/importers/db.py

```diff
@@ -101,9 +101,10 @@ def metadata_dict(dburl, table, extended=None, schema='public'):
 
     metadata, fields = metadata_factory(table.name)
     metadata['objects'][0].update(extended)
-    for col in table.columns:
-        api_name = col.name
-        display_name = col.name
+
+    columns = table.columns if excludes is None else exclude_columns(table.columns, excludes)
```

Comment:
Could we move the excludes=None logic into exclude_columns? seems like it should handle that case itself and would make the callee cleaner.